package.json

```diff
@@ -5,16 +5,16 @@
   "main": "index.js",
   "scripts": {
     "start": "BASE_HREF=/ node server",
+    "test": "npm run unit && npm run lint && npm run build",
```

Comment:
In the description you said you split out `unit` so that `test` only runs other, more focussed tests. I actually prefer unit to still be in here because I think that matches the expectation, but it does contradict your comment!